VMT/VMT.md

```diff
@@ -0,0 +1,225 @@
+| Class B3 | KCSN      | A vulnerability in experimental or debugging features not intended for production use
+| Class C1 | Potential | Not considered a practical vulnerability | (but some people might assign a CVE for it), e.g. one depending on UUID guessing
+| Class C2 | Potential | A vulnerability, but not in Kata Containers supported code, e.g., in a dependency
+| Class D  | Potential | Not a vulnerability, just a bug with (some) security implications, e.g., strengthening opportunities
+| Class E  |           | Neither a vulnerability nor hardening opportunity
+| Class Y  |           | Vulnerability only found in development release
+| Class Z  |           | When due process fails                    |
+
+# KCSA Task status
+
+Here is a summary of the different KCSA task status meanings. These status
+are set in the GitHub Issues using Labels:
+
+| Status        | Meaning                                          |
+| ------------- | ------------------------------------------------ |
+| Incomplete    | It is still unclear whenever the bug warrants an advisory
+| Confirmed     | The vulnerability is confirmed, impact description is in progress
+| Triaged       | Impact description has been submitted for review |
+| In Progress   | CVE has been requested                           |
+| Fix committed | Pre-KCSA has been communicated                   |
+| Fix released  | All patches have been merged                     |
+| Opinion       | Issue is likely a Class B/C/D, waiting for KCSN  |
+| Won't Fix     | Doesn't fit with the project plans, sorry        |
+| Invalid       | Class E and Z, not a bug or vulnerability. No further action to be taken
+
+# Extent of Disclosure
+
+The science of vulnerability management is somewhere around being able to assess impact and severity of a report, being able to design security patches, being an obsessive process-following perfectionist and respecting the rule of lesser disclosure.
+
+Lesser disclosure is about disclosing the vulnerability details to an increasing number of people over time, but only to the people that are necessary to reach the next step.
+The diagram above shows "disclosure extent" across the various steps of the process.
+
+Vulnerability reporters retain final control over the disclosure of their findings.
+If for some reason they are uncomfortable with our process, their choice of disclosure terms prevails.
+
+## Embargo exceptions
+
+> **FIXME** - I'm not sure I fully understand the intent of the below - I need to re-read
+> and check the original...
+
+To keep the embargo period short and effective, the VMT may choose to open Issues.
+Issues that take too much time to be fixed (e.g., more than 2 weeks) or Issues that require a complex patch are usually better solved in the open.
+
+> **FIXME** is the architecture committee the correct group to be consulted here?
```

Comment:
I think so. Flagging that this piece of the process brings us back to the "private issue" issue.